### PR TITLE
feat: inovator badges added

### PR DESCRIPTION
**Pull request by Jaishree2310**
**Description**

This PR fixes #[4808](https://github.com/layer5io/layer5/issues/4808#issuecomment-1742021229)

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by abhijeetgauravm** [comment]
Hey @Jaishree2310 Let's discuss this on website call tomorrow at 5:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

File: assets/badges/innovator/innovator.svg
Comment (ritiksaxena124) [inline assets/badges/innovator/innovator.svg:1]: This won't work. How are you creating SVGs?

File: assets/badges/innovator/innovator.png
Comment (ritiksaxena124) [inline assets/badges/innovator/innovator.png:1]: Suggestions
1. Use dual-tone green for person
2. You can use white for the bulb
3. The person that you are using is different than what we have in other badges. Take a [look](https://badges.layer5.io/) at Bring a Buddy, Sharing is Caring, GitOps with Friends badge

Comment (Jaishree2310) [reply assets/badges/innovator/innovator.png:1]: @ritiksaxena124 there is some issue with the PR. I need to test it once again and will present it in the next meeting.

**Comment by leecalcote** [comment]
Pull request abandoned.

**Pull request closed without merging**